server/flyway/sql/V47__update_role_purpose_length.sql

```diff
@@ -0,0 +1,4 @@
+-- Increase length of role purpose column to 300
+
+ALTER TABLE app_fam.fam_role
+    ALTER COLUMN role_purpose SET DATA TYPE VARCHAR(300);
```

Comment:
found an error while doing the testing, so when grant access with the SPAR role `SPAR_NONMINISTRY_ORCHARD` and a forest client number `00001011`, the role purpose for the new child role `SPAR_NONMINISTRY_ORCHARD_00001011` will be "the role purpose of SPAR_NONMINISTRY_ORCHARD" + "for 00001011", longer than 200 chars. I'm not sure if 500 will be too long? feel 300 is safe for now